boards/default/distros/ubuntu/ubuntu.py

```diff
@@ -0,0 +1,110 @@
+import subprocess as sp
+import shutil
+import pathlib
+import wlutil
+import re
+
+serviceTemplate = """[Unit]
+Requires=multi-user.target
+After=multi-user.target
+Before=firesim.target
+Wants=firesim.target
+
+[Service]
+ExecStart=/etc/firesim/{scriptName} {scriptArgs}
+StandardOutput=journal+console"""
+
+# Some common directories for this module (all absolute paths)
+fed_dir = pathlib.Path(__file__).resolve().parent
```

Comment:
Nit: All `fed` -> `ubu` (similarly all "Fedora" references to "Ubuntu")

Reply:
done